src/duration.rs

```diff
@@ -309,13 +345,22 @@ impl Duration {
     ///
     /// This function errors when duration is less than zero. As standard
     /// library implementation is limited to non-negative values.
+    // TODO: make this method const once our MSRV is 1.58+
     #[cfg(feature = "std")]
     pub fn to_std(&self) -> Result<StdDuration, OutOfRangeError> {
         if self.secs < 0 {
             return Err(OutOfRangeError(()));
         }
         Ok(StdDuration::new(self.secs as u64, self.nanos as u32))
     }
+
+    /// Returns true if and only if d < MIN || d > MAX
```

Comment:
nitpick: why use the mathematical proof phrase "if and only if" here? I suggest wording "if", i.e.

    /// Returns `true` if d < MIN || d > MAX